### DISCUSSION TITLE
【信通院】开源标准研究员招聘

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#38] [日常讨论]
![WeChat Image_20220315145911](https://user-images.githubusercontent.com/62591087/158325046-dc39b405-0ec4-4852-a3f6-7e699ab819fa.jpg)

**Comment by will-ww** [comment]
这个好，解决户口不？:-) @wangleo61

**Reply by wangleo61** [reply]
招聘老师说：我们这个是社招，都是没有户口的，北京的事业单位和企业基本上应届校招才可能会有

**Comment by wangleo61** [comment]
王老师，我问问哈

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: Will ***@***.***>
发送时间: Tuesday, March 15, 2022 9:45 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 【信通院】开源标准研究员招聘 (Discussion #38)


这个好，解决户口不？:-) @wangleo61<https://github.com/wangleo61>

―
Reply to this email directly, view it on GitHub<https://github.com/kaiyuanshe/ONES/discussions/38#discussioncomment-2365210>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5RA3YQRSWOQ2PAU5S5TB3VACH6XANCNFSM5QXYAIUQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by wangleo61** [comment]
王老师，刚才哈，海归未就业可以排队，但是不保证一定能排到，其他的情况解决不了

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

________________________________
发件人: Leo Wang ***@***.***>
发送时间: Tuesday, March 15, 2022 9:45:54 PM
收件人: kaiyuanshe/ONES ***@***.***>; kaiyuanshe/ONES ***@***.***>
抄送: Mention ***@***.***>
主题: 回复: [kaiyuanshe/ONES] 【信通院】开源标准研究员招聘 (Discussion #38)


王老师，我问问哈



从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送



发件人: Will ***@***.***>
发送时间: Tuesday, March 15, 2022 9:45 PM
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [kaiyuanshe/ONES] 【信通院】开源标准研究员招聘 (Discussion #38)



这个好，解决户口不？:-) @wangleo61<https://github.com/wangleo61>

―
Reply to this email directly, view it on GitHub<https://github.com/kaiyuanshe/ONES/discussions/38#discussioncomment-2365210>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AO5RA3YQRSWOQ2PAU5S5TB3VACH6XANCNFSM5QXYAIUQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
